setup.py

```diff
@@ -22,17 +22,30 @@ def write_version_py(filename=None):
         fh.close()
 
 
-write_version_py()
+cfd = os.path.join(
+    os.path.dirname(__file__),
+    'wrappers/CFD', 'version.py'
+)
+dem = os.path.join(
+    os.path.dirname(__file__),
+    'wrappers/DEM', 'version.py'
+)
+
+write_version_py(cfd)
+write_version_py(dem)
 
 setup(
-    name='simkratos',
+    name='wrappers',
```

Comment:
Please also note that for consistency the main folder will also have to be renamed e.g. `wrappers -> simphony_kratos` and the local imports updated as necessary.